pkg/didexchange/invitation/invitation.go

```diff
@@ -0,0 +1,85 @@
+// +build js,wasm
+
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package invitation
+
+import (
+	"encoding/json"
+	"fmt"
+	"syscall/js"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/client/didexchange"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
+)
+
+// RegisterHandleInvitationJSCallback register handle invitation js callback
+func RegisterHandleInvitationJSCallback(didexchangeClient *didexchange.Client) {
```

Comment:
Would it make sense go have a generic registration method for all JS method injections of a package?